app/views/users/logix/profile.html.erb

```diff
@@ -1,27 +1,26 @@
 <% content_for :title, "CircuitVerse - #{@profile.name}" %>
 
-<div class="container">
-  <div class="row feature">
-
     <div class="container">
-      <h2>Profile</h2>
+      <div class="row row-about">
+        <h1 id="heading-profile" class="heading">User Profile</h1>
+      </div>
       <div class="row">
-        <div class="col-lg-8">
-          <p><strong>Name: </strong><%= @profile.name %></p>
-          <p><strong>Member since: </strong><%= @profile.member_since %> days</p>
-          <p><strong>Educational Institution: </strong><%= @profile.educational_institute %></p>
-          <p><strong>Country: </strong><%= @profile.country_name %></p>
-          <p><strong>Subscribed to mails: </strong><%= @profile.mail_subscription %></p>
+        <div class="col-xs col-sm-12 col-md col-lg-4">
+        <%= image_tag @profile.profile_picture.url(:medium), style: "border: 0px; border-radius: 50%; margin: 30px;" %>
+        </div>
+        <div  id="profile-details" class="col-xs col-sm-12 col-md col-lg-7">
+          <h3><%= @profile.name %></h3> <hr>
+          <h4>Member Since</h4>
+          <p><%= @profile.member_since %></p>
+          <h4>Educational Institution</h4>
+          <p><%= @profile.educational_institute %></p>
+          <h4>Country</h4>
+          <p><%= @profile.country_name %></p>
+          <h4>Subscribed to mails ?</h4>
```

Comment:
No question mark needed and capitalisation - `Subscribed To Emails?`

Reply:
@nitin10s whatsay ?